app/angular/src/server/angular-devkit-build-webpack.ts

```diff
@@ -0,0 +1,189 @@
+/**
+ * This file is to be watched !
+ * The code must be compatible from @angular-devkit version 6.1.0 to the latest supported
```

Comment:
Why v6, though? 

https://angular.io/guide/releases#support-policy-and-schedule

```
Angular versions v4, v5, v6, v7, and v8 are no longer under support.
```

Angular does not support those versions, why should we?

Reply:
year but storybook still support these old versions no ? 
I referred here : https://github.com/storybookjs/storybook/blob/ea7d0512d2407dd66a682854fb9cfd8e94057403/app/angular/package.json#L90

Reply:
@shilman @ndelangen What do you think? In the past, I tried to be compatible with the last 4 versions of Angular (8 to 11 right now) 

In the link above you can find a table with LTS date ranges. Should we just stick to what Angular does support and write it somewhere down in the docs so we don't need to ask this ourselves over and over again? 🙂

Reply:
I think we can do it with the next storybook major `7` but not before, no ? 🤔

Reply:
Ideally we keep back compat until our 7.0 release, but I think we can compromise on semver if it helps clean things up a lot.